ci/ci.py

```diff
@@ -343,18 +353,18 @@ def main(argv):
         make_release = True
         engine_artifacts = args.engine_artifacts or "archived"
     elif branch == "editor-dev":
-        engine_channel = "stable"
+        engine_channel = "editor-alpha"
         editor_channel = "editor-alpha"
         release_channel = editor_channel
         make_release = True
         engine_artifacts = args.engine_artifacts
     elif branch and branch.startswith("DEFEDIT-"):
-        engine_channel = None
+        engine_channel = "editor-dev"
```

Comment:
ci.py has three different channel settings:

1. The `engine_channel` setting is used when calling `archive_engine()`, `archive_sdk()` and `archive_bob()` and it is used when deciding where on S3 to upload (ie d.defold.com/archive/CHANNEL/sha1/*).

2. The `editor_channel` setting is used when building and archiving the editor. 

3. The `release_channel` setting is used by the `release()` function, which is called automatically.

The `DEFEDIT-` branch should never really build the engine, but if it does, we don't want build.py to default to `stable` and archive the engine, bob and sdk as a stable release. That's why I chose `editor-dev` as the path. It could perhaps be renamed `engine-dev` though. What matters is that it is a name different from `stable` and that we are able to set a short expiration date for engine, sdk and bob builds uploaded to d.defold.com/archive/editor-dev/sha1/*.

Reply:
Also, since it's called "engine-channel" I associate it with the engine version used/bundled by the editor.